.github/workflows/update-interfaces.yml

```diff
@@ -0,0 +1,41 @@
+name: Update protobuf stubs
+
+on:
+  push
```

Comment:
See my other comment below, but this would need to be adjusted to fully address that.

Reply:
Additionally, since this is only operating on files updated when the submodule changes this should probably be filtered to only run on changes to `api-interfaces`.

Reply:
ooh good call. yeah I saw the trick you used to achieve that in the old CI attempt on api-interfaces, will do the same thing here.